corehq/apps/domain/calculations.py

```diff
@@ -92,7 +91,7 @@ def cases_in_last(domain, days):
             "modified_on": {
                 "from": then,
                 "to": now}}}}
-    data = es_query(params={"domain.exact": domain, 'closed': False}, q=q, es_url=CASE_INDEX + '/case/_search', size=1)
+    data = es_query(params={"domain.exact": domain, 'closed': False}, q=q, es_index='cases', size=1)
```

Comment:
because we need the index key in ES_META and not the actual index name. ES_META could be refactored to use the index name as the key instead.